config/packages/dev/mailer.yaml

```diff
@@ -0,0 +1,5 @@
+framework:
```

Comment:
Do we need to create one for test as well? so that all emails goto a single email address?

Reply:
I'm not sure if this config will use that new style yaml file. But if there is no file in test, the default one is used.

Reply:
Okay